db/migrate/20151206020220_remove_foreign_key_from_users_and_add_foreign_key_to_members.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveForeignKeyFromUsersAndAddForeignKeyToMembers < ActiveRecord::Migration
```

Comment:
Yeah, we weren't sure so we put them in the same migration because they are related. We can split in two, no problem